docs/config.md

```diff
@@ -198,6 +198,17 @@ diff-args = ["--color=always", "$left", "$right"]
 - `$left` and `$right` are replaced with the paths to the left and right
   directories to diff respectively.
 
+By default `jj` will invoke the tool with a directory for the changes.  The
+`--tool-mode` parameter can change this to file by file invocations.  This can
+also be set in the configured as follows:
+
+```toml
+[ui]
```

Comment:
I haven't thoroughly checked, but is this example out-of-date now that the config is per-tool?

Reply:
OK, it's supposed to be

```
[merge-tools.pratdiff]
diff-args = ["--color=always", "$left", "$right"]
mode = "file-by-file"
```

but it's not a good example since the mode doesn't change anything.

Perhaps vimdiff would be a better example. In fact, we could use the file-by-file mode by default there.

Reply:
Switched to vimdiff and renamed mode to `diff-mode`.